src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -38,26 +39,36 @@ namespace ripple {
     This object is created when the consensus process starts, and
     is destroyed when the process is complete.
 */
-class LedgerConsensus
+template <class Traits>
+class LedgerConsensus : public Traits
```

Comment:
Having both `Consensus` and `LedgerConsensus` is confusing. It seems like the difference is in the duration of lifetime (`Consensus` is an application singleton?). Names suggestive of their lifetime would be an improvement. Although I can't think of anything right now.